packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -142,6 +156,12 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
             keybinding: "f2",
             context: NavigatorKeybindingContexts.navigatorActive
         });
+
+        registry.registerKeybinding({
+            command: FileNavigatorCommands.TOGGLE_HIDDEN_FILES.id,
+            keybinding: "i",
```

Comment:
This will not work with the navigator search. Please add a modifier.
![screencast 2018-05-09 09-01-01](https://user-images.githubusercontent.com/1405703/39800389-d7026e04-5367-11e8-8e52-7088fd2d53b2.gif)

Reply:
Fixed. Keybinding was changed to `ctrlcmd+i`.